Core/CoreTests/Login/LoginStartViewControllerTests.swift

```diff
@@ -179,23 +180,34 @@ class LoginStartViewControllerTests: CoreTestCase {
 
     func testQRLoginFeatureGetsTurnedOn() {
         supportsQRCodeLogin = true
-        ExperimentalFeature.qrLogin.isEnabled = false
         controller.view.layoutIfNeeded()
+        controller.viewDidLoad()
         XCTAssertTrue(controller.useQRCodeButton.isHidden)
         XCTAssertTrue(controller.useQRCodeDivider.isHidden)
-        ExperimentalFeature.qrLogin.isEnabled = true
-        XCTAssertFalse(controller.useQRCodeButton.isHidden)
-        XCTAssertFalse(controller.useQRCodeDivider.isHidden)
```

Comment:
This test should be testing what happens when the features gets turned on. Can you add these assertions back and turn 186 into `supportsQRCodeLogin = true` and 181 into `supportsQRCodeLogin = false`?